apps/discovery-platform/migrations/1697029706619_add-configs-table.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable camelcase */
+
+exports.up = (pgm) => {
+  pgm.createTable("configs", {
+    key: { type: "text", primaryKey: true },
+    data: { type: "text", notNull: true },
+    created_at: "createdAt",
+    updated_at: "timestamp",
+  });
+};
+
+exports.down = (pgm) => {
```

Comment:
createTable can automatically infer down migration. you can remove this section entirely.

Reply:
cool